lesson_2/HomeWork_2.py

```diff
@@ -1 +1,4 @@
-test
\ No newline at end of file
+Eleks = ['Rudolf Emir', 'Octav Marango', 'Dominik Toreto', 'Jeccy Tailor', 'Whitny Hyston']
+Toshiba = ['Akida Yamato', 'Yoshida Takad', 'Winhei Huan', 'Triseto Ariso', 'Ritero Naruto', 'Shibao Tikoo']
+Toshiba = Toshiba + Eleks
```

Comment:
As mentioned in the lesson, try not to use + for adding lists. It has a particular function for this.  

Also after your manipulations Eleks should not have employees